checkin/models.py

```diff
@@ -3,6 +3,13 @@
 
 
 class Skill(models.Model):
+    """
+    Skill with image
+
+    :var title: Skill name in Norwegian
+    :var title_en: Skill name in English
```

Comment:
Remove the vars here, they don't add anything. That is, the comment makes as much sense by just having it as 
```python
"""Defines a skill that a user can have."""
```